infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/SnapshotSender.java

```diff
@@ -143,6 +145,8 @@ public void transmit(UUID snapshotSyncEventId) {
             if (completed) {
                 // Block until ACK from last sent message is received
                 try {
+                    // TODO: an improvement : have a resend/increase timeout for ack on the last sent message instead of
```

Comment:
I agree with this comment partially but it is difficult to determine a fixed timeout value for waiting.  How can we say that the current timeout is not enough?  Is there any datapoint for it?

Reply:
We have resends for all but the last msg in snapshot sync. So incase the ACK for the last msg doesn't reach in the first attempt, we CANCEL the sync and retrigger it.
This is unnecessary.

During ITs, its observed that snapshot syncs sometimes gets cancelled and retriggered. 
We either increase the timeout (I don't know how to arrive at the value at this moment, no data) or include resend logic. I think the latter is better. But that would be a non-trivial change and would advocate on having it in a separate PR.